__tests__/pet.test.js

```diff
@@ -0,0 +1,158 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+    it('returns an object', () => {
+        expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+    it('sets the name property', () => {
```

Comment:
Always good to leave a clear line between each of your tests to make it easier to read, should do this throughout all test files here